db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -533,7 +533,6 @@ public void boot(boolean create,Properties startParams) throws StandardException
                     }
                 }
             }
-            startParams.setProperty("catalogVersion", dictionaryVersion.toString());
```

Comment:
Why is this removed?

Reply:
dictionaryVersion is always derby version 10.x. We should set splice build version. It is set somewhere else